crates/ruma-client-api/src/future.rs

```diff
@@ -0,0 +1,38 @@
+//! Endpoints for sending and receiving futures
+
+pub mod future_message_event;
+pub mod future_state_event;
+
+pub mod send_future_update;
```

Comment:
It could be simpler:

```suggestion
pub mod update_future;
```